test/common-algorithms/sorting/parallel/parallel_mergesort.chpl

```diff
@@ -0,0 +1,102 @@
+
+//merges two subarrays
+//First subarray is arr[l..m]
+//Second subarray is arr[m+1..r]
+
+proc merge(arr, left:int, mid:int, right:int){
+	var i: int = 0, j: int = 0, k: int = 0;
```

Comment:
Note that there's a short hand for these declarations:
`var i, j, k: int = 0;` See section 8.1.4 (Multiple Variable Declarations) of the Chapel [spec](http://chapel.cray.com/spec/spec-0.98.pdf).
And that additionally, since 0 is an integer literal value, you don't need to give the type specifier in the first place (though you're certainly welcome to do so!)